### ISSUE TITLE
 dataset  download

### ISSUE THREAD

**Issue by yaolugithub** [alterzero/RBPN-PyTorch#7]
Hi, thanks for your nice code!  
i have a problem of downloading the Vimeo-90k Dataset. i have tried several times to doanload it. however, it did not download the full dataset as it is a very big dataset , thus, could you share your trianing and test set?  
thanks a lot.

**Comment by alterzero** [comment]
As you mentioned, the dataset is huge, and I don't have cloud storage to store that.
Please download it from the official link
http://data.csail.mit.edu/tofu/dataset/vimeo_septuplet.zip

**Comment by yaolugithub** [comment]
Thank you very much.
I have downloaded the training set.
